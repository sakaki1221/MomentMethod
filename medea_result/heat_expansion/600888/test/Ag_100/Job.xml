<?xml version='1.0'?>
<!DOCTYPE MDRESULTS PUBLIC "-//MD//DTD MDRESULTS 1.0" "http://materialsdesign.com/DTD/mdresults_1_0.dtd">

<MDRESULTS>
  <System id="initial">
    <Date>16 Jan 2017 05:52:54 GMT</Date>
    <Characterization>
      <PropertyValue property="symmetry" type="text" units="">P1</PropertyValue>
      <PropertyValue property="setting" type="text" units="">P1</PropertyValue>
      <PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue>
      <PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue>
      <PropertyValue property="primitive Z" type="text" units="">32</PropertyValue>
    </Characterization>
  </System>
  <Calculation id="initial">
    <Date>16 Jan 2017 05:52:54 GMT</Date>
    <System>initial</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-90.471328</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">3.079200</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000
0.000 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">577.500</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-0.078</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">0.07800 0.07800 0.07800 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">0.0400</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-0.8077</PropertyValue>
    </Results>
  </Calculation>
  <System id="displaced_Ag__a">
    <Date>16 Jan 2017 07:37:53 GMT</Date>
    <Characterization>
      <PropertyValue property="symmetry" type="text" units="">P1</PropertyValue>
      <PropertyValue property="setting" type="text" units="">P1</PropertyValue>
      <PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue>
      <PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue>
      <PropertyValue property="primitive Z" type="text" units="">32</PropertyValue>
    </Characterization>
  </System>
  <Calculation id="displaced_Ag__a">
    <Date>16 Jan 2017 07:37:53 GMT</Date>
    <System>displaced_Ag__a</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-90.470438</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">3.079100</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">50979694.608 17031852.621 0.000
1273217853.470 1481539452.150 0.000
1273217907.799 27.165 -1481539452.150
50979695.233 -17031852.621 0.000
-250959134.539 4634517.805 -4634517.720
-81451648.929 0.000 0.000
51211421.125 -17379441.450 0.000
1273217853.470 -27.165 1481539452.150
1273217907.799 -1481539452.150 0.000
50979695.233 0.312 -17031852.621
-250959134.709 4634517.635 4634517.720
-10360927814.894 0.000 0.000
-250959134.370 -4634517.635 -4634517.720
-26648476.890 0.000 0.000
1358145447.163 -1598445161.638 0.000
1358145388.546 -29.308 1598445161.638
51211420.488 17379441.450 0.000
139383120.430 -811040.616 811040.601
-90720684.370 0.000 0.000
50979694.608 -0.312 17031852.621
-250959134.539 -4634517.805 4634517.720
-81451648.929 0.000 0.000
1358145447.163 29.308 -1598445161.638
1358145388.546 1598445161.638 0.000
51211420.488 -0.319 17379441.450
139383120.460 -811040.586 -811040.601
269497205.420 0.000 0.000
139383120.400 811040.586 811040.601
-25953299.232 0.000 0.000
51211421.125 0.319 -17379441.450
139383120.430 811040.616 -811040.601
-90720684.370 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">577.500</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-0.049</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">0.04700 0.04900 0.04900 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">0.0427</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-0.4898</PropertyValue>
    </Results>
  </Calculation>
  <System id="displaced_Ag__b">
    <Date>16 Jan 2017 07:37:54 GMT</Date>
    <Characterization>
      <PropertyValue property="symmetry" type="text" units="">P1</PropertyValue>
      <PropertyValue property="setting" type="text" units="">P1</PropertyValue>
      <PropertyValue property="empirical formula" type="text" units="">Ag</PropertyValue>
      <PropertyValue property="primitive cell formula" type="text" units="">Ag32</PropertyValue>
      <PropertyValue property="primitive Z" type="text" units="">32</PropertyValue>
    </Characterization>
  </System>
  <Calculation id="displaced_Ag__b">
    <Date>16 Jan 2017 07:37:54 GMT</Date>
    <System>displaced_Ag__b</System>
    <Results>
      <PropertyValue property="VASP energy" type="float" units="eV">-90.470438</PropertyValue>
      <PropertyValue property="Fermi energy" type="float" units="eV">3.079100</PropertyValue>
      <PropertyValue dimensions="3 32" property="atomic derivatives" type="float" units="kJ/mol/m">-51095557.543 -17495304.393 0.000
-1358145388.537 -1598908613.410 0.000
-1358145447.171 -29.317 1598908613.410
-51095558.184 17495304.393 0.000
251306723.368 4634517.805 -4634517.720
82030963.644 0.000 0.000
-50979695.233 17031852.621 0.000
-1358145388.537 29.317 -1598908613.410
-1358145447.171 1598908613.410 0.000
-51095558.184 -0.321 17495304.393
251306723.199 4634517.635 4634517.720
10360000911.350 0.000 0.000
251306723.538 -4634517.635 -4634517.720
26532613.947 0.000 0.000
-1273217907.806 1481887040.979 0.000
-1273217853.463 27.171 -1481887040.979
-50979694.608 -17031852.621 0.000
-139614846.316 -695177.671 695177.658
90720684.370 0.000 0.000
-51095557.543 0.321 -17495304.393
251306723.368 -4634517.805 4634517.720
82030963.644 0.000 0.000
-1273217907.806 -27.171 1481887040.979
-1273217853.463 -1481887040.979 0.000
-50979694.608 0.312 -17031852.621
-139614846.290 -695177.645 -695177.658
-270192383.078 0.000 0.000
-139614846.341 695177.645 695177.658
25026395.688 0.000 0.000
-50979695.233 -0.312 17031852.621
-139614846.316 695177.671 -695177.658
90720684.370 0.000 0.000</PropertyValue>
      <PropertyValue property="cell volume" type="float" units="Ang^3">577.500</PropertyValue>
      <PropertyValue property="pressure" type="float" units="GPa">-0.049</PropertyValue>
      <PropertyValue dimensions="6" property="stress" type="float" units="GPa">0.04700 0.05000 0.05000 0.00000 0.00000 0.00000</PropertyValue>
      <PropertyValue property="energy of formation" type="float" units="kJ/mol">0.0427</PropertyValue>
      <PropertyValue property="heat of formation" type="float" units="kJ/mol">-0.4898</PropertyValue>
      <PropertyValue property="zero-point energy" type="float" units="kJ/mol">1.8298</PropertyValue>
      <PropertyValue property="E(0)" type="float" units="kJ/mol">1.8698</PropertyValue>
      <PropertyValue property="E(PV)" type="float" units="kJ/mol">-0.8477</PropertyValue>
      <PropertyValue property="Eelec" type="float" units="kJ/mol">0.0400</PropertyValue>
      <PropertyValue property="Temperatures" type="float" units="K">0.001 1 2 3 4 5 10 15 20 30 40 50 60 70 80 90 100 125 150 175 200 225 250 273 275 298 300 350 400 450 500 550 600 650 700 750 800 850 900 950 1e+003 1.05e+003 1.1e+003 1.15e+003 1.2e+003 1.25e+003 1.3e+003 1.35e+003 1.4e+003 1.45e+003 1.5e+003 1.55e+003 1.6e+003 1.65e+003 1.7e+003 1.75e+003 1.8e+003 1.85e+003 1.9e+003 1.95e+003 2e+003 2.1e+003 2.2e+003 2.3e+003 2.4e+003 2.5e+003 2.6e+003 2.7e+003 2.8e+003 2.9e+003 3e+003</PropertyValue>
      <PropertyValue property="S(T)" type="float" units="J/K/mol">0.0000 0.0000 0.0003 0.0018 0.0048 0.0094 0.0665 0.2419 0.6371 2.1692 4.4020 6.9457 9.5511 12.0919 14.5124 16.7938 18.9347 23.7221 27.8301 31.4040 34.5562 37.3706 39.9097 42.0435 42.2209 44.1775 44.3408 48.1151 51.3998 54.3060 56.9114 59.2720 61.4295 63.4161 65.2567 66.9713 68.5759 70.0837 71.5058 72.8514 74.1282 75.3429 76.5014 77.6084 78.6685 79.6855 80.6626 81.6029 82.5091 83.3836 84.2285 85.0457 85.8369 86.6040 87.3481 88.0707 88.7729 89.4559 90.1207 90.7684 91.3995 92.6160 93.7759 94.8842 95.9455 96.9633 97.9414 98.8825 99.7894 100.6645 101.5100</PropertyValue>
      <PropertyValue property="Cv(T)" type="float" units="kJ/mol">0.0000 0.0000 0.0015 0.0066 0.0152 0.0276 0.1962 0.8160 2.0963 5.8657 9.7705 13.0198 15.5208 17.4022 18.8180 19.8942 20.7237 22.1053 22.9168 23.4288 23.7707 24.0096 24.1828 24.3031 24.3122 24.4044 24.4114 24.5509 24.6420 24.7049 24.7499 24.7834 24.8089 24.8287 24.8445 24.8572 24.8677 24.8763 24.8836 24.8897 24.8950 24.8995 24.9034 24.9068 24.9098 24.9125 24.9148 24.9169 24.9188 24.9205 24.9220 24.9233 24.9246 24.9257 24.9268 24.9277 24.9286 24.9294 24.9301 24.9308 24.9314 24.9326 24.9336 24.9344 24.9352 24.9358 24.9364 24.9369 24.9374 24.9378 24.9382</PropertyValue>
      <PropertyValue property="H(T)-Eelec" type="float" units="kJ/mol">0.9821 0.9821 0.9821 0.9821 0.9821 0.9821 0.9825 0.9848 0.9918 1.0309 1.1094 1.2240 1.3673 1.5324 1.7138 1.9076 2.1109 2.6479 3.2115 3.7912 4.3815 4.9789 5.5814 6.1391 6.1877 6.7480 6.7968 8.0211 9.2511 10.4848 11.7212 12.9596 14.1994 15.4404 16.6822 17.9248 19.1679 20.4115 21.6555 22.8998 24.1444 25.3893 26.6344 27.8796 29.1250 30.3706 31.6162 32.8620 34.1079 35.3539 36.6000 37.8461 39.0923 40.3385 41.5848 42.8312 44.0776 45.3240 46.5705 47.8170 49.0636 51.5568 54.0500 56.5434 59.0369 61.5304 64.0240 66.5177 69.0114 71.5051 73.9989</PropertyValue>
      <PropertyValue property="E(T)-Eelec" type="float" units="">1.8298 1.8298 1.8298 1.8298 1.8298 1.8298 1.8303 1.8325 1.8395 1.8786 1.9572 2.0718 2.2150 2.3801 2.5615 2.7553 2.9586 3.4956 4.0592 4.6390 5.2292 5.8266 6.4291 6.9868 7.0354 7.5957 7.6445 8.8688 10.0988 11.3325 12.5690 13.8073 15.0472 16.2881 17.5299 18.7725 20.0156 21.2592 22.5032 23.7475 24.9922 26.2370 27.4821 28.7273 29.9727 31.2183 32.4640 33.7097 34.9556 36.2016 37.4477 38.6938 39.9400 41.1862 42.4325 43.6789 44.9253 46.1717 47.4182 48.6647 49.9113 52.4045 54.8978 57.3912 59.8846 62.3781 64.8717 67.3654 69.8591 72.3528 74.8466</PropertyValue>
    </Results>
  </Calculation>
</MDRESULTS>
