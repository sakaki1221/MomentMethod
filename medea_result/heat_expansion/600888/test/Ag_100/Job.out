Startup is 'sourced'
Opening the database
Phonon calculation for Untitled

Phonon parameters
=================
The calculation explores the full Brillouin zone and accounts for an interaction range of about 10.0 Angstroem.
The asymmetric atoms are displaced by +/-0.02 Angstroem.

In a first step, need to run an optimization of atom positions for the supercell containing 32 atoms.
Then, need to run a total number of 2 single point energy calculations on supercells with displaced atoms containing 32 atoms.

      Supercell transformation matrix              Supercell lattice vectors
       2.00000    0.00000    0.00000             8.32754    0.00000    0.00000
       0.00000    2.00000    0.00000             0.00000    8.32754    0.00000
       0.00000    0.00000    2.00000             0.00000    0.00000    8.32754

     Supercell           Site        Displaced atom     Displacement (fractional units)
   ---------------    ----------     --------------     -------------------------------
  undisplaced           ----            ----             0.00000    0.00000    0.00000
  displaced_Ag__a       Ag              Ag               0.00240    0.00000    0.00000
  displaced_Ag__b       Ag              Ag              -0.00240    0.00000    0.00000

Invariance conditions are not enforced.
No approximation to LO/TO splitting
Properties: Phonon dispersion and form factors 
            Phonon density of states 
            Thermodynamic functions
==========================================


VASP parameters
===============
Since no magnetic moments are in the model, this is a non-magnetic calculation using 'accurate' precision
and a user-defined planewave cutoff energy of 600.000 eV.

The electronic iterations convergence is 1.00E-005 eV using the Very Fast (RMM-DIIS) algorithm
and reciprocal space projection operators.

The requested k-spacing is 0.1 per Angstrom, which leads to a 8x8x8 mesh.
This corresponds to actual k-spacings of 0.094 x 0.094 x 0.094 per Angstrom.

Using first order Methfessel-Paxton smearing with a width of 0.2 eV.

Other non-default parameters:
	Extrafine augmentation grid for accurate forces is TRUE
	Algorithm is Very Fast (RMM-DIIS)
	Wave functions is TRUE
==========================================

------------------------------------------------------------------------

Using GGA-PBE / PAW potentials:
	Ag

Results for initial structural minimization of the supercell without displacements:
--------------------------------------------------------------------------------------

There are 20 symmetry-unique k-points
The plane wave cutoff is 600.00 eV

   VASP energy:             -90.471328 eV for Ag32

   Initial VASP energy:     -90.471328 eV for Ag32
   Relaxation energy:         0.000000 eV gained after 1 optimization steps.

Electronic contributions:
                          Empirical Formula          Cell
                                    Ag          (Ag)32
                          --------------- ---------------
              VASP Energy        -272.786       -8729.154 kJ/mol
      Energy of formation           0.040           1.281
                  PV term          -0.848         -27.127

   This is the electronic part of the formation energy with respect to the
   elements in their standard state. It is obtained from the total energy (above)
   and tabulated VASP energies for the elements in their standard state.
   See the documentation for more details.

   Details for Ag:
                                             VASP energy (Etot):     -272.79 kJ/mol
       Standard state energy of 1.00 atoms of Ag  1.00 * -272.83:  -  -272.83
                                                                   ----------
                                        Energy of formation (Ef):        0.04 kJ/mol


Cell parameters:
 Parameter    Value    
---------- ------------
         a     8.327538
         b     8.327538
         c     8.327538
     alpha    89.999999
      beta    89.999999
     gamma    89.999999

    Volume    72.187154 Ang^3

        Density:     9.925 Mg/m^3

   The pressure given below is exerted by the system according to its volume.
   Positive pressure would cause expansion during full geometry optimization.
       Pressure:   -78.000 MPa
               =  -780.000 bar

                      XX        YY        ZZ        XY        YZ        ZX
         Stress:    78.000    78.000    78.000     0.000     0.000     0.000 MPa
               =   780.000   780.000   780.000     0.000     0.000     0.000 bar
   The stress tensor above is imposed on the system, i.e. negative values of diagonal components 
   would cause expansion of the corresponding lattice parameter upon full geometry optimization.
   The pressure and stress include only electronic terms, i.e. the vibrational,
   temperature and other terms are not included here.

Fractional Coordinates:
	Atom          Initial Coordinates             Final Coordinates
	-----      -------- -------- --------      -------- -------- --------
	Ag           0.7500   0.7500   0.5000        0.7500   0.7500   0.5000
	Ag           0.7500   0.7500   0.0000        0.7500   0.7500   0.0000
	Ag           0.7500   0.0000   0.2500        0.7500   0.0000   0.2500
	Ag           0.7500   0.2500   0.5000        0.7500   0.2500   0.5000
	Ag           0.0000   0.7500   0.2500        0.0000   0.7500   0.2500
	Ag           0.0000   0.0000   0.5000        0.0000   0.0000   0.5000
	Ag           0.2500   0.7500   0.5000        0.2500   0.7500   0.5000
	Ag           0.7500   0.0000   0.7500        0.7500   0.0000   0.7500
	Ag           0.7500   0.2500   0.0000        0.7500   0.2500   0.0000
	Ag           0.7500   0.5000   0.2500        0.7500   0.5000   0.2500
	Ag           0.0000   0.7500   0.7500        0.0000   0.7500   0.7500
	Ag           0.0000   0.0000   0.0000        0.0000   0.0000   0.0000
	Ag           0.0000   0.2500   0.2500        0.0000   0.2500   0.2500
	Ag           0.0000   0.5000   0.5000        0.0000   0.5000   0.5000
	Ag           0.2500   0.7500   0.0000        0.2500   0.7500   0.0000
	Ag           0.2500   0.0000   0.2500        0.2500   0.0000   0.2500
	Ag           0.2500   0.2500   0.5000        0.2500   0.2500   0.5000
	Ag           0.5000   0.7500   0.2500        0.5000   0.7500   0.2500
	Ag           0.5000   0.0000   0.5000        0.5000   0.0000   0.5000
	Ag           0.7500   0.5000   0.7500        0.7500   0.5000   0.7500
	Ag           0.0000   0.2500   0.7500        0.0000   0.2500   0.7500
	Ag           0.0000   0.5000   0.0000        0.0000   0.5000   0.0000
	Ag           0.2500   0.0000   0.7500        0.2500   0.0000   0.7500
	Ag           0.2500   0.2500   0.0000        0.2500   0.2500   0.0000
	Ag           0.2500   0.5000   0.2500        0.2500   0.5000   0.2500
	Ag           0.5000   0.7500   0.7500        0.5000   0.7500   0.7500
	Ag           0.5000   0.0000   0.0000        0.5000   0.0000   0.0000
	Ag           0.5000   0.2500   0.2500        0.5000   0.2500   0.2500
	Ag           0.5000   0.5000   0.5000        0.5000   0.5000   0.5000
	Ag           0.2500   0.5000   0.7500        0.2500   0.5000   0.7500
	Ag           0.5000   0.2500   0.7500        0.5000   0.2500   0.7500
	Ag           0.5000   0.5000   0.0000        0.5000   0.5000   0.0000

Results for supercell calculations:
------------------------------------------------------------------------
	        Supercell         Energy (eV)
	      -------------      -------------
	        undisplaced       -90.47133
	     displaced_Ag__a      -90.47044
	     displaced_Ag__b      -90.47044


Symmetry analysis of Gamma point phonon frequencies:
----------------------------------------------------
   Crystal  Point  Group: O_h   (m3m) 
   Supercell Point Group: O_h   (m3m) 
    Table of irreducible representations of point group:
 ------------------------------------------
    IR     1  5  7  9 11  6  8 10 12 13 16 17 18 21 23 14 15 19 20 22 24  2  3  4 25 38 39 43 44 46 48 29 30 31 32 33 34 35 36 26 27 28 37 40 41 42 45 47
 ------------------------------------------
 A1g(R)    1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1   x^2+y^2+z^2                         
 A2g       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1                                       
 Eg(R)     2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2  2  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1  2  2  2  0  0  0  0  0  0   (2z^2-x^2-y^2, x^2-y^2)             
 T1g       3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1  3  1  1  1  1  1  1  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 T2g(R)    3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1  3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0 -1 -1 -1  1  1  1  1  1  1   (xy,xz,yz)                          
 A1u       1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1                                       
 A2u       1  1  1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1 -1  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1                                       
 Eu        2 -1 -1 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  0  0  0  0  2  2  2 -2  0  0  0  0  0  0  1  1  1  1  1  1  1  1 -2 -2 -2  0  0  0  0  0  0                                       
 T1u(I)    3  0  0  0  0  0  0  0  0 -1 -1 -1 -1 -1 -1  1  1  1  1  1  1 -1 -1 -1 -3 -1 -1 -1 -1 -1 -1  0  0  0  0  0  0  0  0  1  1  1  1  1  1  1  1  1   (x,y,z)                             
 T2u       3  0  0  0  0  0  0  0  0  1  1  1  1  1  1 -1 -1 -1 -1 -1 -1 -1 -1 -1 -3  1  1  1  1  1  1  0  0  0  0  0  0  0  0  1  1  1 -1 -1 -1 -1 -1 -1                                       
 ------------------------------------------
    Wave vector= 0.0000  0.0000  0.0000    no macro. electric field
 Multi. Omega   Irreducible Representations
  3     0.013     T1u(I)                                                                                                     
Notation: (R)  Raman active mode
          (I)  Infrared active mode
          (RI) Raman and Infrared active mode

Thermodynamic Properties
   for empirical formula Ag

The following thermodynamic functions are obtained from the phonon
density of states within the harmonic approximation. The results are
meaningful only below the melting point, which lies well below 3000 K
for most materials.
Definitions:
      Cv            : vibrational heat capacity at constant volume
E_vib(T)=E(T)-E(0)  : the change in vibrational internal energy from 0 K, where
                        E(0) = E_elec + ZPE is the electronic energy of formation E_elec plus the zero point energy ZPE
      S(T)          : the vibrational entropy at temperature T
   -(A(T)-E(0))     : the change in the vibrational Helmholtz free energy from 0 K
      E(T)          : the electronic energy of formation plus the vibrational internal energy, so this term is
                        E_elec + ZPE + E_vib(T)
      A(T)          : the electronic plus vibrational Helmholtz free energy,
                        E(T) - T.S(T)
Note that the electronic energy, which is part of the enthalpy and free energies,
 is referenced to the elements in their standard state.

Electronic energy =     0.04 kJ/mol (referenced to elements in the standard state)
          PV term =    -0.85 kJ/mol
Zero-point energy =     1.83 kJ/mol

    T       Cv       E(T)-E(0)      S(T)   -(A(T)-E(0))     E(T)        A(T)
    K     J/K/mol      kJ/mol     J/K/mol     kJ/mol       kJ/mol      kJ/mol
  ----- ----------- ----------- ----------- ----------- ----------- -----------
     0      0.0000      0.0000      0.0000      0.0000      1.8698      1.8698
     1      0.0000      0.0000      0.0000      0.0000      1.8698      1.8698
     2      0.0015      0.0000      0.0003      0.0000      1.8698      1.8698
     3      0.0066      0.0000      0.0018      0.0000      1.8698      1.8698
     4      0.0152      0.0000      0.0048      0.0000      1.8698      1.8698
     5      0.0276      0.0000      0.0094      0.0000      1.8698      1.8698
    10      0.1962      0.0005      0.0665      0.0002      1.8703      1.8696
    15      0.8160      0.0028      0.2419      0.0009      1.8726      1.8689
    20      2.0963      0.0098      0.6371      0.0030      1.8796      1.8668
    30      5.8657      0.0488      2.1692      0.0162      1.9186      1.8536
    40      9.7705      0.1274      4.4020      0.0487      1.9972      1.8211
    50     13.0198      0.2420      6.9457      0.1053      2.1118      1.7645
    60     15.5208      0.3853      9.5511      0.1878      2.2551      1.6820
    70     17.4022      0.5503     12.0919      0.2961      2.4201      1.5737
    80     18.8180      0.7318     14.5124      0.4292      2.6016      1.4406
    90     19.8942      0.9256     16.7938      0.5859      2.7954      1.2839
   100     20.7237      1.1288     18.9347      0.7646      2.9986      1.1052
   125     22.1053      1.6658     23.7221      1.2994      3.5356      0.5704
   150     22.9168      2.2294     27.8301      1.9451      4.0992     -0.0753
   175     23.4288      2.8092     31.4040      2.6865      4.6790     -0.8167
   200     23.7707      3.3995     34.5562      3.5117      5.2693     -1.6419
   225     24.0096      3.9969     37.3706      4.4115      5.8667     -2.5417
   250     24.1828      4.5994     39.9097      5.3780      6.4692     -3.5082
   273     24.3031      5.1570     42.0435      6.3208      7.0268     -4.4510
   275     24.3122      5.2056     42.2209      6.4050      7.0755     -4.5352
   298     24.4044      5.7659     44.1775      7.3989      7.6357     -5.5291
   300     24.4114      5.8147     44.3408      7.4874      7.6845     -5.6176
   350     24.5509      7.0391     48.1151      9.8011      8.9089     -7.9313
   400     24.6420      8.2690     51.3998     12.2908     10.1388    -10.4210
   450     24.7049      9.5028     54.3060     14.9348     11.3726    -13.0650
   500     24.7499     10.7392     56.9114     17.7163     12.6090    -15.8465
   550     24.7834     11.9776     59.2720     20.6218     13.8474    -18.7520
   600     24.8089     13.2174     61.4295     23.6401     15.0872    -21.7703
   650     24.8287     14.4583     63.4161     26.7619     16.3281    -24.8921
   700     24.8445     15.7002     65.2567     29.9793     17.5700    -28.1095
   750     24.8572     16.9427     66.9713     33.2854     18.8125    -31.4156
   800     24.8677     18.1859     68.5759     36.6745     20.0557    -34.8047
   850     24.8763     19.4294     70.0837     40.1414     21.2992    -38.2716
   900     24.8836     20.6734     71.5058     43.6814     22.5433    -41.8116
   950     24.8897     21.9178     72.8514     47.2906     23.7876    -45.4208
  1000     24.8950     23.1624     74.1282     50.9654     25.0322    -49.0956
  1050     24.8995     24.4072     75.3429     54.7024     26.2771    -52.8326
  1100     24.9034     25.6523     76.5014     58.4987     27.5221    -56.6289
  1150     24.9068     26.8976     77.6084     62.3516     28.7674    -60.4818
  1200     24.9098     28.1430     78.6685     66.2587     30.0128    -64.3889
  1250     24.9125     29.3885     79.6855     70.2177     31.2583    -68.3479
  1300     24.9148     30.6342     80.6626     74.2265     32.5040    -72.3567
  1350     24.9169     31.8800     81.6029     78.2833     33.7498    -76.4135
  1400     24.9188     33.1259     82.5091     82.3862     34.9957    -80.5164
  1450     24.9205     34.3718     83.3836     86.5336     36.2416    -84.6638
  1500     24.9220     35.6179     84.2285     90.7240     37.4877    -88.8542
  1550     24.9233     36.8640     85.0457     94.9560     38.7338    -93.0862
  1600     24.9246     38.1102     85.8369     99.2281     39.9800    -97.3583
  1650     24.9257     39.3565     86.6040    103.5392     41.2263   -101.6694
  1700     24.9268     40.6028     87.3481    107.8881     42.4726   -106.0183
  1750     24.9277     41.8491     88.0707    112.2736     43.7189   -110.4038
  1800     24.9286     43.0955     88.7729    116.6948     44.9653   -114.8249
  1850     24.9294     44.3420     89.4559    121.1505     46.2118   -119.2807
  1900     24.9301     45.5885     90.1207    125.6400     47.4583   -123.7702
  1950     24.9308     46.8350     90.7684    130.1623     48.7048   -128.2925
  2000     24.9314     48.0815     91.3995    134.7165     49.9513   -132.8467
  2100     24.9326     50.5747     92.6160    143.9177     52.4445   -142.0479
  2200     24.9336     53.0680     93.7759    153.2377     54.9378   -151.3679
  2300     24.9344     55.5614     94.8842    162.6710     57.4312   -160.8012
  2400     24.9352     58.0549     95.9455    172.2128     59.9247   -170.3430
  2500     24.9358     60.5484     96.9633    181.8585     62.4182   -179.9887
  2600     24.9364     63.0420     97.9414    191.6040     64.9118   -189.7342
  2700     24.9369     65.5356     98.8825    201.4455     67.4054   -199.5757
  2800     24.9374     68.0293     99.7894    211.3793     69.8991   -209.5095
  2900     24.9378     70.5231    100.6645    221.4022     72.3929   -219.5324
  3000     24.9382     73.0169    101.5100    231.5110     74.8867   -229.6412

Job completed on Mon 16 January 2017 at 16:40:16 æ±äº¬ (æ¨™æº–æ™‚) after 6442 s (1:47:22)
